llvm/lib/Target/RISCV/RISCVTargetTransformInfo.h

```diff
@@ -250,6 +256,23 @@ class RISCVTTIImpl : public BasicTTIImplBase<RISCVTTIImpl> {
     return ST->is64Bit() && !ST->hasVInstructionsI64();
   }
 
+  bool isLegalStridedLoad(Type *DataType, Align Alignment) {
+    if (!ST->hasVInstructions())
```

Comment:
isLegalStridedLoadStore already exists on TLI.  You can reuse by legalizing the type to a MVT.

Reply:
Ok, will do